packages/augur-sdk/src/state/db/DB.ts

```diff
@@ -351,41 +358,43 @@ export class DB {
-  get ShareTokenBalanceChanged() { return this.dexieDB["ShareTokenBalanceChanged"] as Dexie.Table<ShareTokenBalanceChangedLog, any>; }
-  get Markets() { return this.dexieDB["Markets"] as Dexie.Table<MarketData, any>; }
-  get Dispute() { return this.dexieDB["Dispute"] as Dexie.Table<DisputeDoc, any>; }
-  get CurrentOrders() { return this.dexieDB["CurrentOrders"] as Dexie.Table<CurrentOrder, any>; }
-  get ZeroXOrders() { return this.dexieDB["ZeroXOrders"] as Dexie.Table<StoredOrder, any>; }
+  get CompleteSetsPurchased() { return this.dexieDB['CompleteSetsPurchased'] as Dexie.Table<CompleteSetsPurchasedLog, any>; }
+  get CompleteSetsSold() { return this.dexieDB['CompleteSetsSold'] as Dexie.Table<CompleteSetsSoldLog, any>; }
+  get DisputeCrowdsourcerContribution() { return this.dexieDB['DisputeCrowdsourcerContribution'] as Dexie.Table<DisputeCrowdsourcerContributionLog, any>; }
+  get DisputeCrowdsourcerCompleted() { return this.dexieDB['DisputeCrowdsourcerCompleted'] as Dexie.Table<DisputeCrowdsourcerCompletedLog, any>; }
+  get DisputeCrowdsourcerCreated() { return this.dexieDB['DisputeCrowdsourcerCreated'] as Dexie.Table<DisputeCrowdsourcerCreatedLog, any>; }
+  get DisputeCrowdsourcerRedeemed() { return this.dexieDB['DisputeCrowdsourcerRedeemed'] as Dexie.Table<DisputeCrowdsourcerRedeemedLog, any>; }
+  get DisputeWindowCreated() { return this.dexieDB['DisputeWindowCreated'] as Dexie.Table<DisputeWindowCreatedLog, any>; }
+  get InitialReporterRedeemed() { return this.dexieDB['InitialReporterRedeemed'] as Dexie.Table<InitialReporterRedeemedLog, any>; }
+  get InitialReportSubmitted() { return this.dexieDB['InitialReportSubmitted'] as Dexie.Table<InitialReportSubmittedLog, any>; }
+  get InitialReporterTransferred() { return this.dexieDB['InitialReporterTransferred'] as Dexie.Table<InitialReporterTransferredLog, any>; }
+  get MarketCreated() { return this.dexieDB['MarketCreated'] as Dexie.Table<MarketCreatedLog, any>; }
+  get MarketFinalized() { return this.dexieDB['MarketFinalized'] as Dexie.Table<MarketFinalizedLog, any>; }
+  get MarketMigrated() { return this.dexieDB['MarketMigrated'] as Dexie.Table<MarketMigratedLog, any>; }
+  get MarketParticipantsDisavowed() { return this.dexieDB['MarketParticipantsDisavowed'] as Dexie.Table<MarketParticipantsDisavowedLog, any>; }
+  get MarketTransferred() { return this.dexieDB['MarketTransferred'] as Dexie.Table<MarketTransferredLog, any>; }
+  get MarketVolumeChanged() { return this.dexieDB['MarketVolumeChanged'] as Dexie.Table<MarketVolumeChangedLog, any>; }
+  get MarketOIChanged() { return this.dexieDB['MarketOIChanged'] as Dexie.Table<MarketOIChangedLog, any>; }
+  get OrderEvent() { return this.dexieDB['OrderEvent'] as Dexie.Table<ParsedOrderEventLog, any>; }
+  get Cancel() { return this.dexieDB['Cancel'] as Dexie.Table<CancelLog, any>; }
+  get CancelledOrders() { return this.dexieDB['CancelledOrders'] as Dexie.Table<CancelLog, any>; }
```

Comment:
is the dexie db called `Cancel` or is it called `CancelledOrders` ?

Reply:
`Cancel` is the db of logs from the blockchain. `CancelledOrders` is the derived db. Use `CancelledOrders` since it's indexed on market.